build.gradle

```diff
@@ -150,6 +140,10 @@ project('common') {
         //Do NOT add any additional dependencies here.
     }
 
+    tasks.withType(JavaCompile) {
```

Comment:
nit: Can we have all the compile options at a single place ? ref: https://github.com/pravega/pravega/blob/4f7be177996bd5d3a05e2ab6f117d5a40b1e2207/gradle/java.gradle#L20-L30

`java.gradle `seems to be the place were this change can be added.

Reply:
We could, but we would have to surround the `compileJava` closure with inside a call to `afterEvaluate`. That doesn't seem like a good practice to me, but its doable.